lottieconv/Cargo.toml

```diff
@@ -30,6 +30,7 @@ required-features = ["clap", "webp"]
 default = ["gif", "webp"]
 gif = ["gif_crate"]
 webp = ["webp-animation"]
+serde = ["rgb/serde"]
```

Comment:
Please keep alphabetic ordering of the features (except `default`, that should always be on top)